main.go

```diff
@@ -464,8 +506,8 @@ func main() {
 			Addr:         cfg.server.listen,
 			Handler:      r,
 			TLSConfig:    tlsConfig,
-			ReadTimeout:  readTimeout,  // best set per handler
-			WriteTimeout: writeTimeout, // best set per handler
+			ReadTimeout:  readTimeout,  // best set per handler.
```

Comment:
What does this mean?

Reply:
I guess that's a question for the original author and not Kemal? It means that the timeouts are best set specifically per handler, as the handlers probably have different requirements. /query should for example take only fewer seconds compared to /query_range.